services/getstream/followTopic.js

```diff
@@ -1,6 +1,7 @@
 const stream = require('getstream');
 const {StreamChat} = require('stream-chat');
 const Environment = require('../../config/environment');
+const {CHANNEL_TYPE_STRING} = require('../../helpers/constants');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `CHANNEL_TYPE_STRING` is added but it seems like the `CHANNEL_TYPE` constant is also needed in this file as it's used in line 106. Please ensure that it's imported correctly.
```diff
- const {CHANNEL_TYPE_STRING} = require('../../helpers/constants');
+ const {CHANNEL_TYPE, CHANNEL_TYPE_STRING} = require('../../helpers/constants');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->